src/index.html

```diff
@@ -7,12 +7,64 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="imac_photo"
+      />
+
+      <title class="productNameFont">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </title>
+
+      <div class="productCode">
+        <p class="productCodeFont">Product code: 195434</p>
+      </div>
+
+      <div class="rewiew">
```

Comment:
The class name 'rewiew' is misspelled. It should be 'review' to ensure that the styles are applied correctly.